src/clusterfuzz/_internal/bot/tasks/task_types.py

```diff
@@ -133,7 +134,11 @@ def execute(self, task_argument, job_type, uworker_env):
       return
 
     logs.log('Queueing utask for remote execution.', download_url=download_url)
-    tasks.add_utask_main(command, download_url, job_type)
+    if batch.is_remote_task(command, job_type):
+      tasks.add_utask_main(command, download_url, job_type)
+    else:
+      assert swarming.is_swarming_task(command, job_type)
+      swarming.push_swarming_task(command, download_url, job_type)
```

Comment:
Wont this break fuzz_task, which hashn't been migrated yet?

Reply:
I thought fuzz_task was already migrated. we can disable scheduling fuzz_task on gpu machines until it's migrated, right?

Reply:
Until fuzz_task in migrated, we won't be able to run fuzzing sessions on the gpu machines, however we can still run the other utasks.

This won't affect the existing jobs that are not supposed to run swarming.
For now, the criterion to determine whether a job needs to run on swarming is whether the job has an env variable `REQUIRES_GPU`.